error-prone-contrib/src/test/resources/tech/picnic/errorprone/refasterrules/StringRulesTestInput.java

```diff
@@ -84,4 +84,8 @@ String testSubstringRemainder() {
   int testUtf8EncodedLength() {
     return "foo".getBytes(UTF_8).length;
   }
+
+  String testStringCopyValueOf() {
+    return String.copyValueOf(new char[] {'f', 'o', 'o'}, 0, 3);
+  }
```

Comment:
We generally order the test methods the same as the rules; I'll move these test methods up :)